tensorflow/core/kernels/svd_op_gpu.cu.cc

```diff
@@ -231,7 +216,22 @@ class SvdOpGpu : public AsyncOpKernel {
       DoTranspose(device, outputVT, permAS, outputV);
     }
 
-    done();
+    // Now we check if the SVD operation succeeded or not
+    auto info_checker = [context, dev_info, done](
```

Comment:
Also capture input_copy by value here, since otherwise it might get deallocated before the SVD actually runs, giving you memory corruption. The cost is negligible since "Tensor" is just a light-weight wrapper and copying it only copies a pointer and updates the refcount on the underlying buffer.